mozilla-services/consvc-shepherd
# [SHEP-72] Added flights table to save flights for campaigns

## References

[SHEP-72](https://mozilla-hub.atlassian.net/browse/SHEP-72)

## Problem Statement

Campaigns can have multiple associated flights. To accommodate this, we need to create a separate table for flights.

## Proposed Changes

1. Added a new table for flights. 
2. Updated the existing campaign functionality to utilize the newly created flights table for updating or creating flights associated with campaigns.

## Verification Steps

Create or update campaigns by adding their flight IDs and verify that these are saved in the Flights table for accurate campaigns.

## Visuals

<img width="1415" alt="Screenshot 2024-10-14 at 2 18 36 PM" src="https://github.com/user-attachments/assets/7d4468a2-fdb3-485a-bdb3-b38dd8e5a801">



[SHEP-72]: https://mozilla-hub.atlassian.net/browse/SHEP-72?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
